packages/fluentui/react-northstar-prototypes/src/prototypes/instructionMessage/instructionMessage.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useFluentContext } from '@fluentui/react-bindings';
+import { Box, Ref } from '@fluentui/react-northstar';
+
+interface InstructionMessageProps {
+  message: string;
+  role?: string;
+  'aria-label'?: string;
+}
+
+const InstructionMessage: React.FunctionComponent<InstructionMessageProps> = props => {
+  const { children, message, ...rest } = props;
+
+  const narrationHappened = React.useRef(false);
+  const instructionRef = React.useRef<HTMLElement>();
+  const timeoutMessageNarrate = React.useRef<number>();
+  const timeoutMessageRemove = React.useRef<number>();
+  const context = useFluentContext();
+  const areaFocusableElements = () =>
+    instructionRef?.current?.querySelectorAll('a, button, input, textarea, select, details, [tabindex]');
+
+  const narrate = (instructionMessage, priority = 'polite') => {
+    const element = document.createElement('div');
```

Comment:
Wouldn't it create multiple elements if the user navigates fast enough?

Reply:
I believe so, should we try to get element first if exist ?

Reply:
I didn't see multiple elements created when I'm trying. Because `narrate()` is only called within timeouts. When user quickly navigate, `narrate()` will never be called

Reply:
Actually will not happen because the flag `narrationHappened` is being used to prevent that for what I understood.